doc/status-go-changes.md

```diff
@@ -0,0 +1,26 @@
+# How to make changes to status-go
```

Comment:
I think it would be nice to add the link to this document in the root readme file.